app/middlewares/user_schema.js

```diff
@@ -0,0 +1,27 @@
+const { checkSchema } = require('express-validator');
+const { whiteListDomain } = require('../helpers/white_list_domain');
+
```

Comment:
I think this file would be better in a `schemas` folder

Reply:
I have created a schema folder.